cmd/do-agent/whitelist.go

```diff
@@ -163,3 +163,49 @@ var dbaasWhitelist = map[string]bool{
 
 	"opensearch_http_total_opened": true,
 }
+
+var gpuWhitelist = map[string]bool{
```

Comment:
Do we have temperature sensor values?  I know that was a question from the infra side.